### PR TITLE
Added Company/Shareholders terms to LE vocabulary

### PR DESCRIPTION
**Pull request by samuelashraff**

### CODE REVIEW

**Comment by joakimnordling** [comment]
Please start by resolving the conflicts. You can do that by either merging in the latest master into the branch or then by rebasing the branch on top of the latest master.

**Review by joakimnordling — CHANGES_REQUESTED** [review]
Please start by resolving the conflicts. You can do that by either merging in the latest master into the branch or then by rebasing the branch on top of the latest master.

File: src/pages/Legislation.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import SEO from "../components/seo";
+
+const Legislation = () => (
+    <Layout>
+        <SEO title={"Legislation vocabulary"} />
+        <h1>Life Engine vocabulary - Legislation</h1>
+            <a name={"attribution"}><h2>Attribution</h2></a>
+            <p>Text required to give proper attribution to data source</p>
+            <a name={"name"}><h2>Name</h2></a>
+            <p>Legal name of a company or first name and last name of a person.</p>
```
Comment (joakimnordling) [inline src/pages/Legislation.js]: Is this defintion of a person's name really what's used in legal context; isn't usually the full name used; all first names and the last name? Just thinking aloud, I'm definitely not an expert at all in this field.

Comment (soderluk) [reply src/pages/Legislation.js]: This is for now defined like this, and it has been approved.

**Review by soderluk — APPROVED** [review]

**Review by joakimnordling — APPROVED** [review]
Looks OK to me.

**Pull request merged**